Error in windows setup

my install/run procedure:

$ git clone https://github.com/openlayers/ol-cesium.git
Cloning into 'ol-cesium'...
remote: Counting objects: 5912, done.
remote: Compressing objects: 100% (21/21), done.
remote: Total 5912 (delta 11), reused 15 (delta 5), pack-reused 5886
Receiving objects: 100% (5912/5912), 13.48 MiB | 2.53 MiB/s, done.
Resolving deltas: 100% (3622/3622), done.

$ cd ol-cesium/
$ npm install
npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
added 1001 packages in 185.86s

$ npm run
Scripts available in ol-cesium via npm run-script:
build-examples
TARGET=examples webpack --progress --bail
build-library
TARGET=library webpack --progress --bail
serve
DEV_SERVER=1 TARGET=examples webpack-dev-server --port 3000 --progress --watch --bail --debug

$ npm run build-examples
ol-cesium@2.0.0 build-examples D:\\ol-cesium
TARGET=examples webpack --progress --bail
TARGET is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ol-cesium@2.0.0 build-examples: TARGET=examples webpack --progress --bail
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ol-cesium@2.0.0 build-examples script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\*******\npm-cache\_logs\2018-07-15T20_54_15_847Z-debug.log

Is there a mistake in my procedure or something else ?

--- a/package.json
+++ b/package.json
@@ -3,9 +3,9 @@
   "version": "2.1.0",
   "description": "OpenLayers Cesium integration library",
   "scripts": {
-    "build-examples": "TARGET=examples webpack --progress --bail",
-    "build-library": "TARGET=library webpack --progress --bail",
-    "serve": "DEV_SERVER=1 TARGET=examples webpack-dev-server --port 3000 --progress --watch --bail --debug"
+    "build-examples": "cross-env TARGET=examples webpack --progress --bail",
+    "build-library": "cross-env TARGET=library webpack --progress --bail",
+    "serve": "cross-env DEV_SERVER=1 TARGET=examples webpack-dev-server --port 3000 --progress --watch --bail --debug"
   },
   "main": "./dist/olcesium.js",
   "repository": {
@@ -26,6 +26,7 @@
     "babel-loader": "7.1.5",
     "babel-preset-env": "1.7.0",
     "copy-webpack-plugin": "4.5.2",
+    "cross-env": "5.2.0",
     "css-loader": "1.0.0",
     "eslint": "5.1.0",
     "eslint-config-openlayers": "9.2.0",

